build-data-capturing-gradle-samples/capture-test-pts-support/gradle-test-pts-support.gradle

```diff
@@ -32,7 +32,8 @@ class Capture {
         'org.spockframework.runtime.SpockEngine' : 'spock',
```

Comment:
Will you also be making the same changes for Groovy?

Reply:
Yep, just wanted to nail down the Kotlin implementation, then I'll make the same changes to the Groovy script